### PR TITLE
(Patrick Taylor) I added a short message and my name to the README

### PR DESCRIPTION
**Pull request by ptaylor1021**
I added a short message to the README.

I found it easy, however I did get tripped up briefly when I clicked the "Diff" button and did not see what Prof. Soltoff showed in the video. I did not have a red section showing the contents of the original README, but was still able to successfully upload to Github and create this pull request!

- Patrick Taylor

### CODE REVIEW

**Review by deblnia — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |              |     X       |
| **Coding strategy**         |           |      X        |            |
| **Presentation**            |           |       X       |            |
| **Achievement, creativity** |           |              |  X          |
| **Reproducibility**         |           |    X          |            |

Remarks:

* It seems like you've got a good handle on the git submission workflow, which is great, but the assignment required at least 4 elements of markdown syntax, which you don't have. Be sure to read the assignments carefully going forward!

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # hw01
+
+Hello World! - Patrick Taylor
```
Comment (Ray-RuiTang) [inline README.md:3]: You should introduce a bit more about you and try to use 4 elements of markdown syntax. Anyway, a good start!

**Review by Ray-RuiTang — COMMENTED** [review]
It's nice that you submitted the work successfully! However, showing four or more elements of markdown syntax is required in this homework.

**Comment by maelissvillepontoux** [comment]
Hi Patrick (I was assigned to you,sorry for the late peer review )
Good job on this assignment as you got the basics right.
 Definitely would have been nice to have a photo, link or list in there.

**Review by maelissvillepontoux — COMMENTED** [review]
Hi Patrick (I was assigned to you,sorry for the late peer review )
Good job on this assignment as you got the basics right.
Definitely would have been nice to have a photo, link or list in there.

**Pull request closed without merging**